lib/operations/operation.js

```diff
@@ -39,6 +39,10 @@ class OperationBase {
     delete this.options.session;
   }
 
+  get canRetryRead() {
+    return true;
```

Comment:
why do all operations default to being retryable reads? this should probably be false.

Reply:
It's the nature of the spec. Almost all read operations are retryable, with the exception of `mapReduce`/`getMore`/`runCommand`, and aggregation with a write stage. It would be a ton of boilerplate to make this default to false